deploy-as-code/helm/environments/pspcl-ifix-uat.yaml

```diff
@@ -52,14 +52,15 @@ cluster-configs:
         tracer-errors-provideexceptionindetails: "true"
         timezone: "Asia/Kolkata"
         druid-host: "druid-broker.olap"
+        egov-mdms-host: "https://mgramseva-uat.psegs.in/"
```

Comment:
Remove it from here. Service host names are part of egov-service-host config map.